docs/articles/documentation/test-api/obtaining-data-from-the-client/examples-of-using-client-functions.md

```diff
@@ -0,0 +1,174 @@
+       .click('#submit-button') // Redirects to the 'Thank you' page
+       .expect(getPageUrl()).contains('thank-you', { timeout: 10000 });//Checks if the current page URL contains the 'thank-you' string
+})
+```
+
+### Obtaining a Browser Alias Within a Test
+
+Sometimes you may need to determine a browser alias from within a test. For example, suppose that you configure test tasks by using the [testCafe.createRunner](../../using-testcafe/programming-interface/testcafe.md#createrunner) function and specify several browsers where a test should be run: `runner.browsers['safari','chrome']`. Also suppose that the test logic should be different for different browsers. To implement this, you need to detect which test instance corresponds to `safari` and which one corresponds to `chrome`.
+
+To obtain a browser alias, create a client function that uses the [navigator.userAgent](https://www.w3schools.com/jsref/prop_nav_useragent.asp) property. The function will return a user-agent header sent by the browser. To parse the returned header, you can use external modules, for example, [ua-parser-js](https://github.com/faisalman/ua-parser-js).
+
+The following sample demonstrates how to create conditional test logic based on the used browser. Before running this test, install *ua-parser-js* by running `npm install ua-parser-js`.
+
+```js
+import { ClientFunction } from 'testcafe';
+import uaParser from 'ua-parser-js';
+
+fixture `My fixture`
+   .page `http://example.com`;
+
+//Returns a user-agent header sent by the browser
+const getUA = ClientFunction(() => navigator.userAgent);
+
+test('My Test', async t => {
+    const ua = await getUA();
+    const browserAlias =uaParser(ua).browser.name;
+
+    //Some common actions
+
+    //Conditional logic
+    if (browserAlias === 'Chrome'){
+        //Test logic for Chrome
+        console.log('The browser is Chrome');
+    } else if (browserAlias === 'Safari'){
+        //Test logic for Safari
+        console.log('The browser is Safari');
+    }
+});
+```
+
+## Injecting External Libraries into a Page from a Test
+
+One more case of using client functions is injecting external libraries like [jQuery](https://jquery.com/) into a tested page.
+
+In order to work with jQuery from a test, you first need to add jQuery code to the tested page by using the `eval` method.
+
+```js
+await t.eval(new Function(fs.readFileSync('./jquery.js').toString()));
+```
+
+After that,  you can use jQuery in the test within a client function.
+
+```js
+import { ClientFunction } from 'testcafe';
+import fs from 'fs';
+
+fixture `My fixture`
+   .page `http://devexpress.github.io/testcafe/example/`;
+
+test('My test', async t => {
+    // Adds jQuery code to the page
+    await t.eval(new Function(fs.readFileSync('./jquery.js').toString()));
+
+    const clientFunction = ClientFunction(() => {
+        // Uses the jQuery selector
+        return $('div.column.col-2').text();
+    });
+
+    const text = await clientFunction();
```

Comment:
I offer write assertion: `await t.expect(getElementText()).contains('expectedText');` instead of `console.log`